packages/vscode/src/util.ts

```diff
@@ -25,9 +25,9 @@ export function checkForOtherPrismaExtension(): void {
   if (files.length !== 0) {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     window.showInformationMessage(
-      `You have both the Insider and stable Prisma VSCode extension installed. Please uninstall or disable one of them for a better experience.`,
+      `You have both both versions (Insider and Stable) of the Prisma VSCode extension enabled in your workspace. Please uninstall or disable one of them for a better experience.`,
```

Comment:
```suggestion
      `You have both both versions (Insider and Stable) of the Prisma VS Code extension enabled in your workspace. Please uninstall, or disable, one of them and restart VS Code.`,
```
maybe?
I think a restart is needed after

Reply:
I'm trying to actually get this message to trigger to validate that a restart is required, but I can't seem to get it to pop-up 🤔

Reply:
It can behave weirdly but should be reproducible on a local computer 🤔 (not sure about Gitpod)

Reply:
Trying to dig into this, I think this might warrant a further investigation into the actual logic of this prompt. However, when disabling an extension, vs code should prompt a reload anyway. So it depends on if we want to further specify that already or not

Reply:
So looking at the logic I can understand why it's not working:
On my local computer, there is nothing in `~/.vscode/extensions`
Because I only use the Insiders version.
All the extensions are in `~/.vscode-insiders/extensions`
 
Note: The docs indicate that it's still the correct directory, but don't mention the Insider one:
https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed

Reply:
I only have the `~/.vscode/` folder, do you use the insiders version of VS Code?

Reply:
tracking issue #1335